### ISSUE TITLE
Missing publications Araújo et. al

### ISSUE THREAD

**Issue by ju4nk4** [dealii/publication-list#280]
@article{araujo+wadbro+2021,
author = {Araújo, Juan C. and Wadbro, Eddie},
title = {Shape optimization for the strong directional scattering of dielectric nanorods},
journal = {International Journal for Numerical Methods in Engineering},
volume = {n/a},
number = {n/a},
pages = {},
keywords = {directional scattering, Helmholtz problem, light routing, scattering problem, shape optimization},
doi = {https://doi.org/10.1002/nme.6677},
url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/nme.6677},
eprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/nme.6677},
}

@article{araujo+engstrom+2021,
label         = {A2},
title = "On spurious solutions encountered in Helmholtz scattering resonance computations in {$R^d$} with applications to nano-photonics and acoustics",
journal = {Journal of Computational Physics},
volume = {429},
pages = {110024},
year = {2021},
issn = {0021-9991},
doi = {https://doi.org/10.1016/j.jcp.2020.110024},
url = {https://www.sciencedirect.com/science/article/pii/S0021999120307981},
author = {Juan C. {Araújo C.} and Christian Engstr\"om},
keywords = {Plasmon resonance, Acoustic scattering resonances, Nonlinear eigenvalue problems, Helmholtz problem, Leaky modes, Quasi-normal modes},
}

@article{araujo+engstrom+campos+roman+2019,
label         = {A3},
author = {Juan C. {Araújo C.} and Carmen Campos and Christian Engstr\"om and Jose E. Roman},
title = "Computation of scattering resonances in absorptive and dispersive media with applications to metal-dielectric nano-structures",
journal = "Journal of Computational Physics",
volume = "407",
pages = "109220",
year = "2020",
issn = "0021-9991",
doi = "https://doi.org/10.1016/j.jcp.2019.109220",
url = "http://www.sciencedirect.com/science/article/pii/S0021999119309258",
keywords = "Plasmon resonance, Resonance modes, Nonlinear eigenvalue problems, Helmholtz problem, PML, Dispersion analysis, Leaky modes, Resonant states, Quasimodes, Quasi-normal modes",
}

**Comment by jppelteret** [comment]
Thanks for the contributions!

**Comment by jppelteret** [comment]
May I please clarify, is the first author's full name `Juan C. Araújo C.` (as written in the two 2021 publications) or `Juan C. Araújo` as written in the 2020 publication?

**Comment by ju4nk4** [comment]
Hi, this has been a mistake that I didn't foresee during my PhD studies.
The version I pursue now is
    Juan C. Araújo

I would appreciate if the corresponding publications in deal.II with the
names:
    Juan Carlos Araújo-Cabarcas,
    Juan C. Araújo C,
could also be modified.

Kind regards,

Juan Carlos Araújo Cabarcas, Ph.D.

 "None of us is as clever as all of us" ... Japanese proverb

**Comment by jppelteret** [comment]
I sympathise with you. I have a double-barrelled first name and now also have publications attributed to different variations of my name (or abbreviations of it). Very annoying indeed! (It was because of my own sensitivity to how my name gets listed by the publishers that I noticed the difference in how yours was spelt.) I've made the change that you asked, and it should now be reflected consistently through the publication list.
